README.md

```diff
@@ -102,76 +99,17 @@ terraform apply
 ```
 
 #### Per-user Infrastructure
-* Next build the infrastructure for your user, in the `infrastructure` directory of the Shared Infrastructure project:
-```
-cd infrastructure
-
-# you'll need to init whenever you add new providers in terraform
-# if prompted to migrate all workspaces to S3 then respond with 'yes'
-terraform init -backend-config "bucket=<app_name>-terraform-state"
-# select your workspace
-terraform workspace new <your_workspace>
-terraform apply
-```
+* Next build the infrastructure for your user
+* There is a  `build.sh <app_name> <workspace_name>` script in the `securityanalytics-sharedinfrastructure` directory - run this and it will set up the infrastructure for you. If you are using Windows and Git Bash and it doesn't start that up, see the note above in pre-requisites.
 
 ### Shared Code
-* Now enter the `securityanalytics-sharedcode` directory
-* Use terraform to initalise the infrastructure for this project:
-```
-# setup pipenv (make sure PIPENV_VENV_IN_PROJECT is set to true first)
-pipenv install --dev
-pipenv shell
-
-cd infrastructure
+* Now enter the `securityanalytics-sharedcode` directory and run  `build.sh <app_name> <workspace_name>`
```

Comment:
Can we please add an issue to track that we need a clean up script as well..

Reply:
yes, that was in the back of my mind earlier when I wanted to tear down the test...

Reply:
Let me know the id

Reply:
SA-80